app/styles/components/donations/donation-progress.scss

```diff
@@ -0,0 +1,24 @@
+.donation-progress {
+  .information {
+    display: flex;
+    justify-content: space-between;
+
+    .value {
+      font-weight: 600;
+      font-size: 200%;
+    }
+
+    .descriptive-text {
+      color: gray;
+    }
+
+    .percentage {
+      text-align: right;
+    }
+  }
+
+  .progress-bar-container {
+    margin-top: 10px;
+    height: 10px;
+  }
+}
```

Comment:
There are some alpha-ordering issues here. Would do a general sweep yourself of some of these.